recipes/repeated-test/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "repeated_test" %}
```

Comment:
Another occurrence of issue ( https://github.com/conda-forge/staged-recipes/issues/1073 ).